docs/operations/advanced/troubleshooting.md

```diff
@@ -2,63 +2,36 @@
 
 **Systematic troubleshooting procedures for complex KMS operational issues in 13-party threshold deployments.**
 
-> **Quick Fixes**: For immediate solutions to common problems, see [Common Errors](common-errors.md) and [Quick Reference](../quick-reference.md).
+> **Quick Fixes**: For immediate solutions to common problems, see [Common Errors](common-errors.md) and [Emergency Procedures](../emergency-procedures.md).
 
 ## Systematic Troubleshooting Approach
 
 ### 1. Information Gathering
 
 Before starting troubleshooting, collect the following information:
 
-```bash
-# Kubernetes cluster information
-kubectl version --short
-kubectl get nodes
-kubectl get pods,svc,pvc -n kms-threshold
-
-# Your party's resource utilization
-kubectl top pods -n kms-threshold
-kubectl top nodes
-
-# Network status (production ports)
-kubectl exec -n kms-threshold kms-core-${PARTY_ID} -- \
-  netstat -tlnp | grep -E "(50100|50001|9646)"
-
-# For Docker development environments only
-docker --version && docker ps -a | grep kms
-```
+**System Information**:
```

Comment:
I am wondering if it might make sense to keep the commands for each of these checks? 
Because as I understand developers from all the different teams will have to carry out troubleshooting while on-call. 
And I myself have no experience with kubectl so having the specific commands needed to check certain things would be a great help